pkg/tracing/tracing_test.go

```diff
@@ -113,6 +114,17 @@ func TestPropagateSpan(t *testing.T) {
 	}
 }
 
+func TestPropagateSpanError(t *testing.T) {
+	tracer := NewTracer(mocktracer.New())
+	tracer.inject = func(sm ot.SpanContext, format interface{}, carrier interface{}) error {
+		return errors.New("error")
```

Comment:
The comment at the end of the function says it all...

This is indeed a simple test to ensure that the logging statement in the failure handling code doesn't lead to a crash (which unexercised failure handling path are prone to do)